doc/release-notes.md

```diff
@@ -36,49 +36,429 @@ reindex or re-sync the whole chain.
+rewards will change to 20% and 60% respectively upon activation of the change,
+previous reallocation periods are ignored.
+
+## Enhanced hard fork
+
+Full implementation of the [DIP-0023](https://github.com/dashpay/dips/blob/master/dip-0023.md)
+enhanced hard fork mechanism has been completed. This improved activation
+mechanism includes signaling from both masternodes and miners while also
+ensuring sufficient upgrade time is available to partners. This new system is
+enabled by the `v20` activation and `SPORK_24_EHF` spork (__TODO__: is this true?) and will be used for
+the `mn_rr` hard fork.
+
+## Asset lock and unlock transactions
+
+The asset lock process provides a mechanism for locking and unlocking Dash in a
+credit pool. This credit pool mirrors the available credit supply in Dash
+Platform. When a certain amount of Dash is locked, a proportional number of
+credits are issued to the identity specified by the transaction issuer. The
+identity then uses the credits to pay fees for operations done on Dash
+Platform. Masternode identities receive the fees and can convert them back to
+Dash by withdrawing them from Dash Platform. Please refer to [DIP-0027](https://github.com/dashpay/dips/blob/master/dip-0027.md) (__TODO__: NOT MERGED)
+for more information.
+
+Note: The asset lock feature is activated by the `v20` hard fork while the asset
+unlock feature is activated by the `mn_rr` hard fork.
+
+## Coinbase Changes
+
+Once v20 activates, Coinbase transactions in all mined blocks must be of
+version 3.
+
+Version 3 of Coinbase will include the following three fields:
+- `bestCLSignature` (`BLSSignature`) will hold the best Chainlock signature
+  known at the moment
+- `bestCLHeightDiff` (`uint32`) is equal to the diff in heights from the mined
+  block height
+- `creditPoolBalance` (`int64`) is equal to all the duff locked in all previous
+  blocks minus all the duff unlocked in all previous blocks plus all the block
+  subsidy paid to evonodes on Platform.
 
-This release fixes a couple of issues with mixing on nodes that start with no
-wallet loaded initially.
+Although miners are forced to include a version 3 Coinbase, the actual
+real-time best Chainlock signature isn't required as long as blocks contain a
+`bestCLSignature` equal to (or newer than) the previous block.
 
-## Wallet GUI improvements
+Note: Until the first `bestCLSignature` is included in a Coinbase once v20
+activates, null `bestCLSignature` and `bestCLHeightDiff` values are perfectly
+valid.
 
-Wallets with 100k+ txes should now be able to rescan without hanging forever
-while processing notifications for every tx. Running `keypoolrefill` with a
-large number of keys will no longer lockup the GUI and can be interrupted.
-Running `upgradetohd` can also be interrupted now.
+## Testnet Breaking Changes
 
-## Changes in RPCs, commands and config options
+A new testnet only LLMQ has been added. This LLMQ is of the type `LLMQ_25_67`,
```

Comment:
Looks like this LLMQ was actually in v19 (https://github.com/dashpay/dash/pull/5225, https://github.com/dashpay/dash/blob/master/src/llmq/params.h#L449)

Reply:
🙈